libmamba/src/core/output.cpp

```diff
@@ -267,6 +267,13 @@ namespace mamba
     {
     public:
 
+        ConsoleData(const Context& ctx)
+            : context(ctx)
+        {
+        }
+
+        const Context& context;
```

Comment:
There is a Clang-tidy check for that, will look into setting that up!

Reply:
@AntoinePrv I think the clang-tidy rule forces the same pattern for all members, be they private or public, whatever the type - if that's the case then we cant use that as it's not the norm everywhere in this code and would break public members in many classes.
If it does just follow the pattern existing in a class, then that would work.